cg/meta/orders/fastq_submitter.py

```diff
@@ -55,27 +56,30 @@ def order_to_status(order: OrderIn) -> dict:
         }
         return status_data
 
-    @staticmethod
-    def _get_fastq_pipeline(
-        application_version: models.ApplicationVersion, is_tumour: bool
-    ) -> Pipeline:
-        if is_tumour:
-            return Pipeline.FASTQ
-        if application_version.application.prep_category == "wgs":
-            return Pipeline.MIP_DNA
-
-        return Pipeline.FASTQ
+    def create_maf_case(self, sample_obj: models.Sample):
+        case_obj: models.Family = self.status.add_case(
+            data_analysis=Pipeline(Pipeline.MIP_DNA),
+            data_delivery=DataDelivery(DataDelivery.ANALYSIS_FILES),
+            name="_".join([sample_obj.name, "MAF"]),
+            panels=None,
+            priority=Priority.research,
+        )
+        case_obj.customer = self.status.customer("cust000")
```

Comment:
```suggestion
        case_obj.customer = self.status.customer(MAF_CUSTOMER_ID)
```

Reply:
How about declaring `production_customer = self.status.customer("cust000")` in the function and then using it? I think `MAF_CUSTOMER_ID` can create confusion.